### ISSUE TITLE
Project Status?

### ISSUE THREAD

**Issue by djensen47** [Ullfis/aurelia-mdc-bridge#46]
Hi @Ullfis ,

I was hoping to find a way to contact you outside of Github but there's no email nor social media on your Github profile so ... I'm just wondering, what's the status of this project? 

I appreciate all the past work you've done but it's been a few months since the last update. So I just want to see if you're still around and interested in the project and what the plans are for the future.

Feel free to shoot me a message (see my profile) or respond here.

Thanks,
Dave

**Comment by djensen47** [comment]
I'm currently discussing creating an org and a fork with @dragoscirjan

**Comment by djensen47** [comment]
Any updates? Is this project no longer supported?

**Comment by jasonhjohnson** [comment]
@djensen47 Have you started your fork of this yet? I'd love to use it but it seems largely based on legacy MDC versions

**Comment by djensen47** [comment]
I have not started a fork yet and it looks like @dragoscirjan abandoned his efforts as well.

I'm still surprised nobody else has taken this on.

**Comment by jasonhjohnson** [comment]
@djensen47 Yeah would be super helpful and seems like there would be more demand

**Comment by dragoscirjan** [comment]
@djensen47 @jasonhjohnson Haven't abandoned yet, just that I have too much work to do and too little time to do everything. I'm trying to limit the number of side-projects I work on, in parallel, otherwise, I wouldn't get to sleep at all :)).

Maybe we could start a chat on Gitter/Slack/etc, see what ideas we have, what's missing from this project and try to share the effort in bringing it back to life. I started my own 'fork' (it's quite a different project completely) because I have no knowledge of Typescript (willing to learn) and I also have a different vision on how this project should be... packed.

**Comment by jasonhjohnson** [comment]
@dragoscirjan I just added you to the gitter channel.

**Comment by djensen47** [comment]
@jasonhjohnson Which gitter channel?

**Comment by jasonhjohnson** [comment]
https://gitter.im/aurelia-material-design-components-bridge/community
